ephios/core/consequences.py

```diff
@@ -88,7 +88,7 @@ class WorkingHoursConsequenceHandler(BaseConsequenceHandler):
     def create(
         cls,
         user: UserProfile,
-        when: datetime,
+        when: date,
```

Comment:
is this breaking existing consequences?

Reply:
this is just a change in type annotation. types are not enforced at runtime, and from what I saw, we are already only passing dates. Also, on the model, it's a date field.